plugin/trino-exasol/src/main/java/io/trino/plugin/exasol/ExasolClient.java

```diff
@@ -262,6 +286,74 @@ private static LongWriteFunction dateWriteFunctionUsingSqlDate()
         });
     }
 
+    private static ColumnMapping hashTypeVarBinaryColumnMapping()
+    {
+        return ColumnMapping.sliceMapping(
+                VarbinaryType.VARBINARY,
+                hashTypeReadFunction(),
+                hashTypeWriteFunction());
+    }
+
+    private static SliceReadFunction hashTypeReadFunction()
+    {
+        return (resultSet, columnIndex) -> {
+            String hex = resultSet.getString(columnIndex);
+            byte[] bytes = hexStringToByteArray(hex);
+            return Slices.wrappedBuffer(bytes);
+        };
+    }
+
+    private static SliceWriteFunction hashTypeWriteFunction()
+    {
+        return SliceWriteFunction.of(Types.CHAR, (statement, index, slice) -> {
+            byte[] bytes = slice.getBytes();
+            String hex = byteArrayToHexString(bytes);
+            statement.setString(index, hex);
+        });
+    }
+
+    // Converts a hexadecimal string (e.g. "0A1B2C") into a byte array.
+    private static byte[] hexStringToByteArray(String hexString)
+    {
+        // Get the number of characters in the hex string.
+        int stringLength = hexString.length();
+
+        // Each byte is represented by 2 hex characters, so allocate half as many bytes.
+        byte[] byteArray = new byte[stringLength / 2];
+
+        // Process two characters (1 byte) at a time.
+        for (int charIndex = 0; charIndex < stringLength; charIndex += 2) {
```

Comment:
Did you implement the logic from scratch, or could you provide a reference/link if available?